scripts/ci/configure.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+((${BASH_VERSION%%.*} >= 4)) || { echo >&2 "$0: Error: Please upgrade Bash."; exit 1; }
+
+set -euxo pipefail
+
+conan --version # check it works
+
+cmake_args=(
+    -S .
+    -B build
+    -G "${AUDACITY_CMAKE_GENERATOR}"
+    -D audacity_use_pch=no
+    -D audacity_has_networking=yes
+    -D CMAKE_INSTALL_PREFIX="${DEST}"
+)
+
+if [[ "${OSTYPE}" == msys* ]]; then # Windows
+
+    cmake_args+=(
+        -A "${AUDACITY_CMAKE_PLATFORM}"
+    )
+
+elif [[ "${OSTYPE}" == darwin* ]]; then # macOS
+
+    cmake_args+=(
+        -T buildsystem=1
+    )
+
+else # Linux & others
+
+    cmake_args+=(
+        # none
```

Comment:
for non-multiconfiguration generators build type should be set during configuration step:
-D CMAKE_BUILD_TYPE=Release